File: notes-application/NotesApp.Api/Models/Note.cs
namespace NotesApp.Api.Models
{
    public class Note
    {
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string? Content { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public int UserId { get; set; }
    }
}

File: notes-application/notesapp-frontend/src/types/note.ts
export interface NotePayload {
  title: string;
  content: string;
}

export interface Note {
  id: number;
  title: string;
  content: string;
  createdAt: string;
  updatedAt: string;
}


export interface PaginatedNoteResponse {
  items: Note[]
  totalItems: number
  page: number
  limit: number
}